{{ cookiecutter.project_name }}/Cartfile.private

```diff
@@ -1,2 +1,2 @@
-github "Quick/Quick" "swift-3.0"
-github "Quick/Nimble" "master"
+github "Quick/Quick" "v0.10.0"
```

Comment:
Can we use version pinning for these instead of pointing at a tag? `~> 0.10` here and `~> 5.0` below for Nimble.